CHANGELOG/v1.7.0.md

```diff
@@ -0,0 +1,433 @@
+- CI: Add fail fast to DumpResourcesForCluster in case of no route to host (#10204)
+- CI: Add more templating func to prowjob-gen (#10403)
+- CI: Added go directive test (#10261)
+- CI: Bump conversion-gen to v0.29.0 (#10012)
+- CI: Bump go-apidiff to v0.8.2 (#10011)
+- CI: Bump govulncheck to v1.0.4 (#10274)
+- CI: Bump kpromo to v4.0.5 (#10140)
+- CI: Bump kubebuilder envtest to 1.29.0 (#10013)
+- CI: Change base branch for link checker (#10365)
+- CI: DumpResourcesForCluster should fail fast for i/o errors (#10238)
+- CI: Ensure build images contains correct binary and for correct architecture (#9932)
+- CI: Fix Make target generate-go-openapi (#10161)
+- CI: Fix TestPatchNode flake (#10287)
+- CI: Hack/prowjob-gen skip creating empty files (#10022)
+- CI: implement generator for prowjobs (#9937)
+- CI: Makefile bump dependencies (#10236)
+- CI: bump WorkloadKubernetesVersion for v1.6 clusterctl upgrade test (#10017)
+- CI: adjust capd dockerfile so the binary exposes the package in the built binraries path variable (#10030)
+- CI: Update GH actions to work with new release-1.6 branch (#9708)
+- ClusterClass: Add ClusterClass variables metadata (#10308)
+- ClusterClass: Add unit tests for MachinePools for webhooks and cc controller (#10055)
+- ClusterClass: Add unit tests for MachinePools in topology/scope package (#10052)
+- ClusterClass: Implement topology validations for topology kubernetes version upgrades (#10063)
+- ClusterClass: use the alias for ClusterCacheTrackerReader instead of the internal reference (#10309)
+- clusterctl: Add 0 default to worker-machine-count help (#10203)
+- clusterctl: Add completion for fish shell (#9950)
+- clusterctl: Add hivelocity infra provider to clusterctl (#10168)
+- clusterctl: Add in-cluster ipam provider (#8811)
+- clusterctl: Add Proxmox provider quickstart (#9798)
+- clusterctl: Better verbose logging on override path (#10180)
+- clusterctl: Bump cert-manager to 1.14.2 (#10126)
+- clusterctl: Bump cert-manager to 1.14.4 (#10267)
+- clusterctl: Bump cert-manager to v1.14.1 (#10113)
+- clusterctl: Clarify rules for adding new clusterctl default providers (#9975)
+- clusterctl: Deprecate clusterctl alpha topology plan (#10139)
+- clusterctl: Replace context.TODO() from clusterctl proxy.go (#9776)
+- Community meeting: Move CecileRobertMichon to emeritus (#10042)
+- Community meeting: Move ykakarap to emeritus approvers (#10000)
+- Community meeting: Promote chrischdi to Cluster API maintainer (#9997)
+- Dependency: Bump `github.com/docker/docker` from 24.0.7 -> 25.0.0 (#10057)
+- Dependency: Bump `golangci-lint` to v1.56.1 (#10124)
+- Dependency: Bump controller-runtime to v0.17.1 (#10131)
+- Dependency: Bump controller-runtime to v0.17.2 (#10162)
+- Dependency: Bump controller-tools to v0.14 (#9987)
+- Dependency: Bump crypto dep version (#9938)
+- Dependency: Bump github.com/onsi/ginkgo/v2 from 2.16.0 to 2.17.0 (#10284)
+- Dependency: Bump github.com/onsi/gomega from 1.31.1 to 1.32.0 (#10285)
```

Comment:
optional: already part of below dependencies section? propably matches more of these `Dependency: *` parts?

Reply:
Agree. Suggested to drop them below